<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Detalles_Aparta</class>
 <widget class="QWidget" name="Detalles_Aparta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>551</width>
     <height>291</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>5</number>
   </property>
   <widget class="QWidget" name="tab_1">
    <attribute name="title">
     <string>Aparta A1</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>170</y>
       <width>111</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Desea alquilarlo ?</string>
     </property>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Si</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>No</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_A1">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Ver info</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_A1">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>321</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Aparta A2</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_A2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Ver info</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_A2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>331</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>180</y>
       <width>111</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Desea alquilarlo ?</string>
     </property>
     <widget class="QRadioButton" name="radioButton_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Si</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>No</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Aparta A3</string>
    </attribute>
    <widget class="QTextBrowser" name="textBrowser_A3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>331</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_A3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Ver info</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>180</y>
       <width>111</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Desea alquilarlo ?</string>
     </property>
     <widget class="QRadioButton" name="radioButton_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Si</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_16">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>No</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Aparta A4</string>
    </attribute>
    <widget class="QTextBrowser" name="textBrowser_A4">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>331</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_A4">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Ver info</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_9">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>180</y>
       <width>111</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Desea alquilarlo ?</string>
     </property>
     <widget class="QRadioButton" name="radioButton_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Si</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_18">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>No</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Aparta A5</string>
    </attribute>
    <widget class="QTextBrowser" name="textBrowser_A5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>331</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_A5">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Ver info</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_10">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>180</y>
       <width>111</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Desea alquilarlo ?</string>
     </property>
     <widget class="QRadioButton" name="radioButton_19">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Si</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_20">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>No</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>Aparta A6</string>
    </attribute>
    <widget class="QTextBrowser" name="textBrowser_A6">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>331</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_A6">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Ver info</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_11">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>180</y>
       <width>111</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Desea alquilarlo ?</string>
     </property>
     <widget class="QRadioButton" name="radioButton_21">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Si</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_22">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>No</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lobster Two</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Lista de Apartamentos</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>303</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
